commands-yml/commands/device/recording-screen/start-recording-screen.yml

```diff
@@ -0,0 +1,118 @@
+---
+name: Start Record Screen
+short_description: Returns the information of the system state which is supported to read as like cpu, memory, network traffic, and battery
+
+example_usage:
+  java:
+    |
+      driver.startRecordingScreen();
+      driver.startRecordingScreen(new BaseStartScreenRecordingOptions(....));
+  javascript_wd:
+    |
+      await driver.startRecordingScreen();
+  ruby:
+    |
+      @driver.start_recording_screen
+      @driver.start_recording_screen video_size: '1280x720', time_limit: '180', bit_rate: '5000000' # Android
+      @driver.start_recording_screen video_type: 'h264', time_limit: '260' # iOS
+  python:
+    |
+      self.driver.start_recording_screen()
+  php:
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://static.javadoc.io/io.appium/java-client/6.1.0/io/appium/java_client/screenrecording/CanRecordScreen.html#startRecordingScreen-T-"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3412"
+  ruby: "https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Android/Device#start_recording_screen-instance_method"
```

Comment:
Why is this in Android? Doesn't it work for iOS too?

Reply:
Yes, it works both platform. Ruby has 2 URLs for each platform. But generating documentation command raised an error then. I'll take a look at the script and update this line/or the script.